wikimedia_publish.py

```
@@ -2,639 +2,206 @@
-    
-    # Create separate payloads for each content variant group
+
+    for filename, info in files.items():
+        variant = get_content_variant(filename)
+        file_groups.setdefault(variant, []).append((filename, info))
+
     payloads = []
-    
-    for content_variant, file_list in file_groups.items():
+
+    for variant, file_list in file_groups.items():
```

Ruff (0.14.14)

[warning] Loop control variable `variant` not used within loop body

(B007)